ExpantaNum.js

```diff
@@ -277,6 +302,10 @@
   Q.isInfinite=function (x){
     return new ExpantaNum(x).isInfinite();
   };
+  P.setInfinite=function (){
+    this.array[0][1]=Infinity;
+    this.standardize();
+  }
```

Comment:
This mutates `this`. Why is this needed? Also, it is probably better to just do `this.array=[[0,Infinity]]` and skip `standardize`.